src/utils/getErrorMessage.js

```diff
@@ -0,0 +1,14 @@
+const getMessage = (error) => {
+  let message;
+  if (error.response.data.message === undefined) {
+    const data = error.response.data;
+    const keys = Object.keys(data);
+    message = data[keys[0]];
+  }
+  else {
+    message = error.response.data.message;
+  }
+  return message;
+}
+
+export default getMessage;
```

Comment:
use destruction in these cases, for example:

const { data } = error.response

....